docs/en/compute/Tensorflow.md

```diff
@@ -64,12 +72,14 @@ section.
 
 ## Examples: Image Recognition
 
+### Preparing training data
+
 If the training data is already in a remote data storage, you can mount it as a folder under 
 the Alluxio `/training-data` directory. Those data will be visible to the applications running on
 local `/mnt/fuse/`.
 
-For example, we suppose the ImageNet data is stored in an S3 bucket `s3a://alluxio-tensorflow-imagenet/` 
-We can mount it into path `/training-data/imagenet` by running:
+Suppose the ImageNet data is stored in a S3 bucket `s3a://alluxio-tensorflow-imagenet/`.
```

Comment:
I remember `s3a://alluxio-tensorflow-imagenet/` is not a public / read-only bucket. Can you double check?
otherwise you can use the `s3a://alluxio-quick-start/tensorflow`

Reply:
I just think we provide a real bucket with real data, that's good because users can acutally try out the example more quickly, but it requires additional setups:
```
$ bin/alluxio fs mount /training-data s3a://alluxio-quick-start/tensorflow \
--share --option alluxio.underfs.s3a.inherit_acl=false \
--option aws.accessKeyID=<ACCESS_KEY_ID> \
--option aws.secretKey=<SECRET_KEY>
Mounted s3a://alluxio-quick-start/tensorflow at /training-data

$ bin/alluxio fs chown <USER_NAME>:<GROUP_NAME> -R /training-data
```
In the official docs, I will prefer to give more general usage: users can create their s3 bucket which is private and they have the write access.